mytest/test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
this is also the wrong test structure. You should be testing using the ktest makefile scripts that are used by the other tests in the directory I mentioned above.